curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comment-your-javascript-code.md

```diff
@@ -13,21 +13,20 @@ Comments are lines of code that JavaScript will intentionally ignore. Comments a
 
 There are two ways to write comments in JavaScript:
 
-Using `//` will tell JavaScript to ignore the remainder of the text on the current line:
+Using `//` will tell JavaScript to ignore the remainder of the text on the current line. This is an in-line comment:
 
 ```js
 // This is an in-line comment.
```

Comment:
Will these not be translated? If they will be, then I do not see the reason for the addition.

Reply:
They are in code blocks, so cannot be translated through Crowdin. When/if we update the comments dictionary to parse the description and instruction fields, these could be handled that way. But for now, there is no translation flow for them.